src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -126,6 +126,13 @@ private InternalXrefSpec[] AggregateXrefSpecs(string uid, InternalXrefSpec[] spe
                 return specsWithSameUid;
             }
 
+            // loc override fallback uid
+            if (specsWithSameUid.Length == 2 &&
```

Comment:
consider use filtering and not return directly here, there are more filtering below like moniker check.